components/EmotionForm.js

```diff
@@ -79,20 +96,28 @@ export default function EmotionForm({ onSubmit, defaultValue, onCancel }) {
               onChange={handleChangeEmotionType}
             >
               <option value="">---Choose an Emotion---</option>
-              {emotionList.map((emotion) => (
-                <option key={emotion.emotionType} value={emotion.emotionType}>
+              {emotionTypes.map((emotion) => (
+                <option key={emotion.id} value={emotion.emotionType}>
                   {emotion.emotionType}
                 </option>
               ))}
             </StyledSelectEmotion>
             <StyledArrow>▼</StyledArrow>
           </SelectEmotionContainer>
-
+          {!defaultValue && (
+            <StyledCreateEmotionLink
+              href="/createemotiontype"
+              aria-label="Create a new emotion type"
+            >
+              Create your Emotion Type <MoodPlusIcon />
+            </StyledCreateEmotionLink>
+          )}
           <label htmlFor="intensity">Emotion intensity*</label>
           <SliderIntensity
             emotionType={selectedEmotionType}
             defaultIntensity={selectedIntensity}
             onChange={(intensity) => setSelectedIntensity(intensity)}
+            emotionTypes={emotionTypes}
```

Comment:
instead of passing down the emotionType and emotionTypes, you can find the selectedEmotionType here in this component and only pass that down to the slider.